### DISCUSSION TITLE
question: inpaint

### DISCUSSION THREAD

**Discussion by kalle07** [Acly/krita-ai-diffusion#761] [Q&A]
hey there,

you can handle real SDXL-Inpaint models ?
Or you have your own way like Fooocus and only need a normal SDXL_Model
i see only one strenght slider, in A1111 it give also one slider but with 6 different options (fill,original,latent, - maske,whole image)

**Comment by Acly** [comment]
Works like in Fooocus, checkpoints are patched on the fly.  Pre-merged inpaint models are not supported.

https://github.com/Acly/krita-ai-diffusion/wiki/Inpainting

**Reply by kalle07** [reply]
i see ...
and the other options for tune the inpaint ? are you working on that?

**Reply by Acly** [reply]
there are lots of options, what are you missing specifically?

**Reply by kalle07** [reply]
like i said   ;)

![grafik](https://github.com/Acly/krita-ai-diffusion/assets/118767589/20b3799d-dcdc-41c2-ab66-e877ed233249)
